api/status/redis-cluster_status.go

```diff
@@ -14,6 +14,7 @@ const (
 	RedisClusterInitializing RedisClusterState = "Initializing"
 	RedisClusterBootstrap    RedisClusterState = "Bootstrap"
 	// RedisClusterReady means the RedisCluster is ready for use, we use redis-cli --cluster check 127.0.0.1:6379 to check the cluster status
-	RedisClusterReady  RedisClusterState = "Ready"
-	RedisClusterFailed RedisClusterState = "Failed"
+	RedisClusterReady    RedisClusterState = "Ready"
+	RedisClusterFailed   RedisClusterState = "Failed"
+	RedisClusterDegraded RedisClusterState = "Degraded"
```

Comment:
To simplify things for now, I suggest not adding any additional statuses. We can just use the existing "failed" status.